source/concepts/protocols.rst

```diff
@@ -0,0 +1,310 @@
+.. SPDX-FileCopyrightText: 2021 Hyperledger
+   SPDX-License-Identifier: CC-BY-4.0
+
+*********
+Protocols
+*********
+
+Perun state channel protocols are a set of protocols for setting up a state
+channels, doing off-chain transactions on these channels and settling them.
+
+The life cycle of a state channel consists of 4 phases:
+
+1. **Open phase**: Setup up the channel and deposit funds.
+2. **Transact phase**: Do off-chain transactions.
+3. **Register phase**: Agree on what state should be settled.
+4. **Settle phase**: Settle the channel and withdraw funds.
+
+Types of state channels
+=======================
+
+Perun protocols support three types of channels:
+
+1. Ledger channel.
+2. Sub-channel.
+3. Virtual channel.
+
+Life cycle for each of these channels consist of the same four phases described
+above. The differences are in how the channels are funded during the ``open
+phase`` and, how they are settled. This layer, where funds will be deposited /
+channels will be settled will be referred to as ``parent layer`` in the further
+discussions.
+
+Ledger channel
+--------------
+
+   - Formed between any two parties having sufficient funds in their on-chain
+     accounts.
+   - The funds are directly deposited into the smart contracts on the
+     blockchain.
+   - Allows any number of off-chain transactions between the participants.
+   - The channel must settled on the blockchain.
+
+Sub-channel
+-----------
+
+   - Formed between participants who have a ledger channel established between
+     them.
+   - Funds are locked on the ledger channel. Hence, no on-chain interaction
+     required to setup a sub-channel.
+   - Sub-Channel is settled,
+
+     - by unlocking the funds from the ledger channel, if both parties agree on
+       the final state of the channel through off-chain transactions; or
+     - by settling the parent channel on the blockchain, otherwise.
+
+       In this case, the parent channel and all other sub-channels opened using
+       the parent channel will also be settled on the blockhain.
```

Comment:
To which indenting does this section belong to? To the subsection or to both? If to subsection, then add it drectly afterwards, otherwise the intendation should be corrected.

Reply:
It belongs to the subsection. I have moved it directly afterwards. 655746a

Reply:
ok